app/controllers/api_controller/providers.rb

```diff
@@ -5,9 +5,14 @@ module Providers
     CREDENTIALS_ATTR  = "credentials"
     AUTH_TYPE_ATTR    = "auth_type"
     DEFAULT_AUTH_TYPE = "default"
-    ENDPOINT_ATTRS    = %w(hostname ipaddress port)
+    ENDPOINT_ATTRS    = %w(hostname ipaddress port connection_configurations_array).freeze
     RESTRICTED_ATTRS  = [TYPE_ATTR, CREDENTIALS_ATTR, ZONE_ATTR, "zone_id"]
 
+    def show_providers
+      @additional_attributes = %w(connection_configurations_array)
+      show_generic(:providers)
+    end
+
```

Comment:
do we really want this for all queries,  we could just ask for it as needed, i.e. GET /api/providers?expand=resources&attributes=connection_configurations_array  (as long as connection_configurations_array is declared as a virtual attribute)